specs/core/0_beacon-chain.md

```diff
@@ -922,6 +922,18 @@ def get_new_validator_registry_delta_chain_tip(current_validator_registry_delta_
     )
 ```
 
+#### `get_domain`
+
+```python
+def get_domain(fork_data: ForkData,
```

Comment:
I moved `get_domain` under Helper functions since it's getting used by `BLSVerify`

Reply:
Could you add a corresponding entry to the table of contents?